Mathlib/Algebra/Polynomial/Laurent.lean

```diff
@@ -335,6 +335,9 @@ theorem _root_.Polynomial.toLaurent_inj (f g : R[X]) : toLaurent f = toLaurent g
 theorem _root_.Polynomial.toLaurent_ne_zero {f : R[X]} : f ≠ 0 ↔ toLaurent f ≠ 0 :=
   (map_ne_zero_iff _ Polynomial.toLaurent_injective).symm
 
+theorem _root_.Polynomial.toLaurent_eq_zero {f : R[X]} : f = 0 ↔ toLaurent f = 0 :=
```

Comment:
If you swap the two sides of the `iff`, then this could be tagged `simp`.
I guess the lemma above this one should also be flipped.